migrate/012_hugepages.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:vm_host) do
+      add_column :total_hugepages_1g, :text, null: false, default: 0
+      add_column :used_hugepages_1g, :text, null: false, default: 0
```

Comment:
I think they are integer
```suggestion
      add_column :total_hugepages_1g, Integer, null: false, default: 0
      add_column :used_hugepages_1g, Integer, null: false, default: 0
```